Unaccountable-Magpie/BangazonAPI2
# paymenttype controller done

Purpose:
Creates the api calls for getting payment type info


How it fits in context of the project:
gets the info for payment types


Specific Feature affected:

controllers

How to test (Be thorough!):

1.checkout to my branch jr-controller

2. open PaymentTypesController

3. Press IIS Express play button to start up the server, a window will open inside the browser. You will need to add /api/paymenttypes to end of the url in order to access my controller. Copy this address as you will need to paste inside Postman

4. Open PostMan, and select GET

5. Copy that address from the browser inside there and press SEND

6. Select POST within PostMan and use something like this 
 {
        "name": "Cash",
       "AccountNumber": 78478,
      "CustomerId": 2,
"IsDeleted": "True"
    }

7. add a new object

8. Rerun the GET and you should see a new object

9. Select PUT within PostMan use something like this to change something in the object and put the id # in the url 
 {
        "Name": "cash",
      "CustomerId": 2
    }
10 .Update one of the objects

11. Rerun the GET and you should see the updated object

12. Select DELETE within PostMan

13. Delete one of the objects

14. Rerun the GET and you shouldn't see that object because it was deleted
